pkg/services/ngalert/state/manager.go

```diff
@@ -68,9 +68,8 @@ func NewManager(logger log.Logger, metrics *metrics.State, externalURL *url.URL,
 	return manager
 }
 
-func (st *Manager) Close(ctx context.Context) {
+func (st *Manager) Close() {
 	st.quit <- struct{}{}
```

Comment:
I know it wasn't changed in this pull request, but I think this should be `close(st.quit)`? Perhaps we could fix it in a follow up change.

Reply:
I will open a PR